packages/scan-request-sender/src/sender/dispatcher.ts

```diff
@@ -13,10 +14,11 @@ export class Dispatcher {
         @inject(PageDocumentProvider) private readonly pageDocumentProvider: PageDocumentProvider,
         @inject(Logger) private readonly logger: Logger,
         @inject(ScanRequestSender) private readonly sender: ScanRequestSender,
+        @inject(ServiceConfiguration) private readonly serviceConfig: ServiceConfiguration,
     ) {}
 
     public async dispatchScanRequests(): Promise<void> {
-        const configQueueSize = Number(process.env.QUEUE_SIZE);
+        const configQueueSize = (await this.serviceConfig.getConfigValue('commonConfig')).maxQueueSize;
```

Comment:
the api call should be serviceConfig.getMaxQueueSize() or something similar.
getConfigValue('commonConfig') --> this shows that user of the class should aware of the section of the config.

Reply:
i didnt want to create a method for every property that exists in the config. So, user can get a property under a group object like scanConfig / commonConfig etc.